quilt/tools/build.py

```diff
@@ -333,3 +334,17 @@ def generate_build_file(startpath, outfilename=DEFAULT_BUILDFILE):
     with open(buildfilepath, 'w') as outfile:
         yaml.dump(contents, outfile, default_flow_style=False)
     return buildfilepath
+
+def clone_git_repo(git_url, destination):
+    cmd = ['git', 'clone', '-q', '--depth=1', git_url, destination]
+    try:
+        proc = subprocess.Popen(cmd, cwd=os.getcwd(), env=os.environ.copy())
```

Comment:
I think those are the defaults for `cwd` and `env`, so don't need them?

Reply:
Also, I think `subprocess.call` or `subprocess.check_call` is what you want - instead of calling `wait`, etc.